projects/storefrontapp-e2e-cypress/cypress/integration/regression/update-password.e2e-spec.ts

```diff
@@ -0,0 +1,67 @@
+import * as helper from '../../helpers/login';
+
+describe('My Account - Update Password', () => {
+  const PAGE_TITLE_UPDATE_PASSWORD = 'Update Password';
+  const PAGE_TITLE_HOME = 'Homepage';
+  const PAGE_URL_UPDATE_PASSWORD = '/my-account/update-password';
+  const PAGE_URL_LOGIN = '/login';
+
+  describe('when an anonymous user tries to access the page', () => {
+    it('should be redirected to the login page', () => {
+      cy.visit(PAGE_URL_UPDATE_PASSWORD);
+      cy.url().should('contain', PAGE_URL_LOGIN);
+    });
+  });
+
+  describe('when a user is authenticated', () => {
+    let user: any;
+    const newPassword = 'newPassword123!';
+
+    before(() => {
+      cy.visit('/');
+      user = helper.registerUser();
+      helper.signOutUser();
+    });
+
+    beforeEach(() => {
+      cy.visit('/');
+      helper.loginWithCredentials(user.email, user.password);
+      cy.selectUserMenuOption('Password');
+      cy.url().should('contain', PAGE_URL_UPDATE_PASSWORD);
+      cy.title().should('eq', PAGE_TITLE_UPDATE_PASSWORD);
```

Comment:
Why do you need to assert  in `beforeEach`?

Reply:
Then Password item in the My Account menu is one of the feature we need to test.  Moreover, if the test setup fails, we need the early assertion to reduce investigation time.